AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -39,6 +39,11 @@ def open_level(directory : str, level : str):
 
         :return: None
         """
+        # Make sure we are not in game mode
+        if general.is_in_game_mode():
+            general.exit_game_mode()
+            TestHelper.wait_for_condition(lambda : not general.is_in_game_mode(), 1.0)            
```

Comment:
After the "wait for condition" is finished probably worth still checking if it's in game mode, and if it is then fail.

Reply:
will do